lib/ome/common/module.cpp

```diff
@@ -76,9 +76,9 @@ namespace
     // Global paths (not specific to any component)
     static RegisterModule bin
       ("bin",
-       "BIOFORMATS_BINDIR",
+       "OME_FILES_BINDIR",
```

Comment:
Note that my initial comment was mainly related to the `OME_FILES_HOME` environment variable, below, so may not have been addressing your question properly.,  `OME_FILES_BINDIR` is certainly out of place here.  (Sorry, but I still have a terrible migraine and didn't notice the line the comment was on properly.)